libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1111,4 +1118,18 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         }
     }
+
+    protected void setAttributeValuesIfNotDefault(AztecAttributes attributes, MediaFile mediaFile) {
+        if (!mediaFile.getTitle().equalsIgnoreCase(DEFAULT_MEDIA_TITLE)) {
```

Comment:
Sometimes the media file's title contains the filename of the image, and we probably don't want to use that as the `title` attribute.

![screenshot_1493501602](https://cloud.githubusercontent.com/assets/3903757/25559074/202f733e-2d02-11e7-8be0-1ee0d1ac9a18.png)